data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -263,7 +263,10 @@
         {RECALL_XY Dolburras 23 11} # he's slow, so put him near the bridge
         {RECALL_XY Grug 23 11}
 
+#ifndef EASY
+        #on Easy, Dacyn starts with full hitpoints in the previous scenario, so it would be odd for him to be injured here
         {MODIFY_UNIT (id=Dacyn) hitpoints 21}
```

Comment:
As with previously, as a fan of increased inter-difficulty variance, might I suggest instead: `{ON_DIFFICULTY 47 34 21}`

Reply:
Done!